### PR TITLE
Rename parameters

### PR DESCRIPTION
**Pull request by gassmoeller**
I think its better to rename these parameters now instead of waiting until everyone got used to them. They have not been in any Release yet, do we need an entry in changes.h for that?

### CODE REVIEW

**Comment by bangerth** [comment]
Are we using "Minimum" instead of "Minimal" elsewhere, or why do you want to change this?

**Comment by gassmoeller** [comment]
Its sister plugin 'Uniform radial' uses "Minimum ..." and "Maximum ..." for coordinate input parameters and we have a "Minimum refinement function" and "Maximum refinement function" plugin. 
On the other hand a quick search for minimal showed also some use (there is a "minimal_temperature" function for the boundary temperature).
Originally I thought "Minimum" is more usual in English and me using "Minimal" in this context was only a German heritage, is that not the case?

**Comment by bangerth** [comment]
I don't know. "minimal" is an adjective that modifies the following noun. "minimum temperature" is a combination of two nouns.

Do we have any native speakers getting these emails?

**Comment by spco** [comment]
Randomly sticking my nose in, I'd say both minimum and minimal are legitimate adjectives in normal English. A *highly scientific* Google search suggests that the technical difference is whether you have a partial or full ordering - if you have only a partial ordering then you can only have a minimal element, not a minimum element. As the reals have a full ordering, the definitions coincide anyway. That fits with usual usage.

TLDR; both are probably fine in this context. ;)

**Comment by bangerth** [comment]
@spco -- you're not helping here ;-)

**Comment by bangerth** [comment]
Let's just go with @gassmoeller 's suggestion then. I think it follows what @spco says, and it seems to be more in line with uses elsewhere in the code base.
